plugins/BEdita/API/src/Controller/AppController.php

```diff
@@ -55,7 +55,11 @@ public function initialize()
 
         $this->response = $this->response->withHeader('X-BEdita-Version', Configure::read('BEdita.version'));
 
-        $this->loadComponent('BEdita/API.Paginator');
+        $pagination = Configure::read('Pagination');
+        if (empty($pagination)) {
+            $pagination = [];
+        }
+        $this->loadComponent('BEdita/API.Paginator', $pagination);
```

Comment:
(This is a _veeeeery minor_ issue…)

In order to reduce complexity (which makes Scrutinizer complain at times) we could avoid the `if` statement by just writing:
```php
$this->loadComponent('BEdita/API.Paginator', (array)Configure::read('Pagination'));
```

If that configuration key is missing, `Configure::read()` returns `null`, and casting NULL to an array results in an empty array (`(array)null === []`), which is perfect. 😎